ChaosMod/Effects/db/Misc/MiscCocktail.cpp

```diff
@@ -0,0 +1,49 @@
+//
+// effect by veloscocity
+//
+
+#include "Memory/Gravity.h"
+#include <math.h>
+#include <stdafx.h>
```

Comment:
```suggestion
#include <stdafx.h>

#include "Memory/Gravity.h"
```
`stdafx.h` needs to be the first include to not cause issues. Also `math.h` should be added there instead.